website/src/components/articles/article-content-footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+interface Props {
+  lastUpdated: string;
+  lastAuthorName: string;
+}
+
+export const ArticleContentFooter: FC<Props> = ({
+  lastUpdated,
+  lastAuthorName,
+}) => {
+  return (
+    <Footer>
+      <Small>
```

Comment:
A link to the git history to this file would be probably nice. What do you think?

Reply:
I can't follow. What exactly do you mean?

Reply:
Sorry, I mean it would be also cool to have an additional link for every markdown document (perhaps placed next to the last document change date), which points directly to document's commit history. Does this make more sense 😄